scripts/install.sh

```diff
@@ -119,17 +119,24 @@ apiVersion: kubelet.config.k8s.io/v1beta1
 kind: KubeletConfiguration
 shutdownGracePeriod: 30s
 shutdownGracePeriodCriticalPods: 10s
----
 EOF
     else
         cat << EOF >> $KUBEADM_CONF_FILE
 apiVersion: kubelet.config.k8s.io/v1beta1
 kind: KubeletConfiguration
 cgroupDriver: systemd
----
 EOF
     fi
 
+    # conditional kubelet configuration fields
+    if [ -n "$CONTAINER_LOG_MAX_SIZE" ]; then
+        echo "containerLogMaxSize: $CONTAINER_LOG_MAX_SIZE" >> $KUBEADM_CONF_FILE
+    fi
+    if [ $CONTAINER_LOG_MAX_FILES -gt 0 ]; then
```

Comment:
Is it possible this is unset? If so bash will print an error.

Reply:
i tried it both providing a value and not, seemed to work fine. i guess it just defaults to zero since that's the behavior in Go. i will validate it once more to make sure.

Reply:
nevermind, you are right